test/dbservice_web/controllers/enrollment_record_controller_test.exs

```diff
@@ -24,25 +40,25 @@ defmodule DbserviceWeb.EnrollmentRecordControllerTest do
   describe "index" do
     test "lists all enrollment_record", %{conn: conn} do
       conn = get(conn, Routes.enrollment_record_path(conn, :index))
-      assert json_response(conn, 200)["data"] == []
+      [head | _tail] = json_response(conn, 200)
+      assert head["academic_year"] == "2017-18"
     end
   end
 
   describe "create enrollment_record" do
     test "renders enrollment_record when data is valid", %{conn: conn} do
-      conn =
-        post(conn, Routes.enrollment_record_path(conn, :create), enrollment_record: @create_attrs)
-
-      assert %{"id" => id} = json_response(conn, 201)["data"]
+      conn = post(conn, Routes.enrollment_record_path(conn, :create), @create_attrs)
+      assert %{"id" => id} = json_response(conn, 201)
```

Comment:
is this right syntax? I believe `json_response` returns a data. So it should be assignment operation. How come `assert` will work here? Which two values assert will compare?